src/tls/openssl.c

```diff
@@ -312,6 +318,101 @@ static int windows_load_system_certificates(struct tls_context *ctx)
 }
 #endif
 
+#ifdef __APPLE__
```

Comment:
This could be enough for us but it's just a my curiosity -- How come using __APPLE__ instead of FLB_SYSTEM_MACOS here?

Reply:
__APPLE__ gets included on mac via vscode, whereas FLB_SYSTEM_MACOS doesn't. I can use that.

Reply:
Ah, this is correct. `__APPLE__` is only defined for apple products but Darwin is only indicated that that OS just uses Darwin kernel.